### DISCUSSION TITLE
[Feature]:  Option to Support MergingMedia Source for React Native Video.

### DISCUSSION THREAD

**Discussion by afkcodes** [TheWidlarzGroup/react-native-video#4970] [Ideas]
### What problem are you trying to solve?

We have a use case where someone has the audio and the video file separated, but they need to play simultaneously in sync. This would give more flexibility for different use cases where separate audio and video files/remote URLs need to be played in sync.

### Proposed solution

Add support for `MergingMediaSource` to React Native Video, so separate audio and video sources can be merged and played simultaneously in sync.

### Platforms

Android

### Additional context

Firstly thanks for this amazing lib.

**Possible implementation:** I do not have it but here is a doc: https://developer.android.com/reference/androidx/media3/exoplayer/source/MergingMediaSource

**Code sample:** Not Available

**Comment by afkcodes** [comment]
Any update on this ?

**Comment by freeboub** [comment]
> Any update on this ?

No there is no planning for this feature yet

**Comment by afkcodes** [comment]
Hi @freeboub any plans for this ? the only thing that is stopping me from building my small app.

**Comment by moskalakamil** [comment]
Hey @afkcodes, since this feature is not planned to be right now in the roadmap, if this is important to you and you need it, please contact [TheWidlarzGroup](https://thewidlarzgroup.com/) - [hi@thewidlarzgroup.com](mailto:hi@thewidlarzgroup.com) for commercial support

**Comment by TWG-AI** [comment]
Thanks for the idea — noted for the team to consider.

> I'm an AI assistant, so double-check any specifics I may have mentioned.
